src/content/en/2019/page-weight.md

```diff
@@ -6,809 +6,765 @@ description: Page Weight chapter of the 2019 Web Almanac covering why page weigh
 
-**HTML** – HTML is typically the smallest resource on the page. Its performance risk is negligible. 
+### Bigger, complex pages can be bad for your business
 
-**Images** – Unoptimized images are often the greatest contributor to page bloat. Looking at the 90th percentile of the HTTP Archive data gathered for the Almanac, images accounted for a whopping 5220KB of a roughly 7MB page. In other words, images comprised almost 75% of the total page weight. The number of images on a page has been linked to lower conversion rates on retail sites. (More on that later.)
+Let's assume you're not a heartless monster who doesn't care about your site's visitors. But if you are, you should know that serving bigger, more complex pages hurts you, too. That was one of the findings of a [Google-led machine-learning study](https://www.thinkwithgoogle.com/marketing-resources/experience-design/mobile-page-speed-load-time/) that gathered over a million beacons worth of real user data from retail sites.
 
-**JavaScript** – JavaScript matters. A page can have a relatively low JS weight but still suffer from JS-inflicted performance problems. A single 100KB third-party script can wreak havoc with your page. The more scripts on your page, the greater the risk. 
+There were three really important takeaways from this research by Google:
 
-It's not enough to focus solely on blocking JS. It's possible for your pages to contain zero blocking resources and still have less-than-optimal performance because of how your JavaScript is rendered. That's why it's so important to understand CPU usage on your pages – because JavaScript consumes more CPU than all other browser activities combined. While JS blocks the CPU, the browser can't respond to user input. This creates what’s commonly called “jank” – that annoying feeling of jittery, unstable page rendering. 
+1. **The total number of elements on a page was the greatest predictor of conversions.** Hopefully this doesn't come as a huge surprise to you, given what we've just covered about the performance risks imposed by the various assets that make up a modern web page.
 
-**CSS** – Stylesheets are an incredible boon for modern web pages. They solve a myriad of design problems, from browser compatibility to design maintenance and updating. Without stylesheets, we wouldn’t have great things like responsive design. But, like JavaScript, CSS doesn’t have to be bulky to cause problems. Poorly executed stylesheets can create a host of performance problems, ranging from stylesheets that take too long to download and parse, to improperly placed stylesheets that block the rest of the page from rendering. And like JS, more CSS files equals more potential trouble.
+2. **The number of images on a page was the second greatest predictor of conversions.** Sessions which converted users had 38% fewer images than sessions that didn't convert.
 
+<figure>
```

Comment:
Does this interfere with the list numbering?

Reply:
I don't believe it should because <ol> is a block level element. But we might have to manually use `<ol>` and `<li>` here if the markdown bugs out.